App.js

```diff
@@ -11,23 +15,56 @@ import rootReducer from './reducers'
 
 const store = createStore(rootReducer, composeWithDevTools(applyMiddleware(thunk)))
 
+const AppNavBar = createBottomTabNavigator(
+  {
+    Home: HomeScreen,
+    Profile: ProfilePage,
+    Messages: MessagesScreen,
+    Schedule: ScheduleScreen
+  },
+  {
+    defaultNavigationOptions: ({ navigation }) => ({
+      tabBarIcon: ({ focused, horizontal, tintColor }) => {
+        const { routeName } = navigation.state;
+        let iconName;
+        if (routeName === 'Home') {
+          iconName = 'home'
+        } else if (routeName === 'Profile') {
+          iconName = 'user'
+        } else if (routeName === 'Messages') {
+          iconName = 'message-circle'
+        } else if (routeName === 'Schedule') {
+          iconName = 'calendar'
+        }
+
+        return <Icon name={iconName} size={25} color={tintColor} />;
+      },
+    }),
+    tabBarOptions: {
+      activeTintColor: '#4AA9C5',
+      inactiveTintColor: 'gray',
```

Comment:
I really like the active tint color! It's subtle and fits with the palette well.